### PR TITLE
[IOTDB-787] set thrift and antlr as two independent module

### PR DESCRIPTION
**Pull request by jixuan1989**
As discussed in https://lists.apache.org/thread.html/rd776647235c16c67db732e104a7eedda8e7b654d2d368462e586adf6%40%3Cdev.iotdb.apache.org%3E

 
We will extract the thrift.thrift and antlr.g4 into two independent modules. 

After that, just use `mvn generate-source -pl thrift,antlr` or `mvn compile -pl thrift,antlr` to generate codes.

**OR:**

** JUST RIGHT CLICK the project on IDEA, and CHOOSE `maven -> Generate Sources and Update Folders`**

Enjoy it!

### CODE REVIEW

File: antlr/src/pypi/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+Using the package, you can write data to IoTDB, read data from IoTDB and maintain the schema of IoTDB.
+
+## Requirements
+
+You have to install thrift (>=0.13) before using the package.
+
+## How to use (Example)
+
+You can get an example of using the package to read and write data at here: [Example](https://github.com/apache/incubator-iotdb/blob/release%2F0.9.0/client-py/src/client_example.py)
```
Comment (qiaojialin) [inline antlr/src/pypi/README.md:52]: 0.9.0？

File: session/src/test/java/org/apache/iotdb/session/CompleteFutureTest.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.session;
+
+
+
+public class CompleteFutureTest {
```
Comment (qiaojialin) [inline session/src/test/java/org/apache/iotdb/session/CompleteFutureTest.java:24]: remove

File: antlr/src/pypi/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+# Apache IoTDB
```
Comment (qiaojialin) [inline antlr/src/pypi/README.md:22]: Do we need this in antlr module?

File: jdbc/src/test/java/org/apache/iotdb/jdbc/manualtest/JDBCRegisterManyTimeSeries.java [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Statement;
+import org.apache.iotdb.jdbc.IoTDBSQLException;
+
+public class JDBCRegisterManyTimeSeries {
+
+  public static void main(String[] args) throws ClassNotFoundException, SQLException {
```
[the remark this answers is not in the document]
Comment (jixuan1989) [reply jdbc/src/test/java/org/apache/iotdb/jdbc/manualtest/JDBCRegisterManyTimeSeries.java:50]: Will remove this..

File: jdbc/src/test/java/org/apache/iotdb/jdbc/manualtest/CompleteFutureTest.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.jdbc.manualtest;
+
+
+
+public class CompleteFutureTest {
```
[the remark this answers is not in the document]
Comment (jixuan1989) [reply jdbc/src/test/java/org/apache/iotdb/jdbc/manualtest/CompleteFutureTest.java:24]: Will remove this...

**Pull request merged**